apps/web/src/components/forms/edit-document/add-title.action.ts

```diff
@@ -0,0 +1,17 @@
+'use server';
+
+import { updateTitle } from '@documenso/lib/server-only/document/update-title';
+import type { TAddTitleFormSchema } from '@documenso/ui/primitives/document-flow/add-title.types';
+
+export type AddTitleActionInput = TAddTitleFormSchema & {
+  documentId: number;
+};
+
+export const addTitle = async ({ documentId, title }: AddTitleActionInput) => {
+  'use server';
```

Comment:
Need to check for userId here otherwise I could theoretically update anyone's document

Reply:
Yeah 👍